cedar-policy-core/src/entities/json/value.rs

```diff
@@ -614,45 +581,31 @@ impl<'e> ValueParser<'e> {
 #[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize)]
 #[serde(untagged)]
 pub enum EntityUidJson {
-    /// Explicit `__expr` escape; see notes on `CedarValueJson::ExprEscape`.
-    ///
-    /// Deprecated since the 1.2 release; use
-    /// `{ "__entity": { "type": "...", "id": "..." } }` instead.
+    /// Explicit `__expr` syntax.
+    /// This is no longer supported and is only here for generating nice error messages.
     ExplicitExprEscape {
-        /// String to interpret as a (restricted) Cedar expression.
-        /// In this case, it must evaluate to an entity reference.
-        __expr: SmolStr,
+        /// Contents are ignored.
+        __expr: String,
     },
-    /// Explicit `__entity` escape; see notes on `CedarValueJson::EntityEscape`
+    /// Explicit `__entity` escape; see notes on JSONValue::EntityEscape
     ExplicitEntityEscape {
         /// JSON object containing the entity type and ID
         __entity: TypeAndId,
     },
-    /// Implicit `__expr` escape, in which case we'll just see a JSON string.
-    ///
-    /// Deprecated since the 1.2 release; use
-    /// `{ "type": "...", "id": "..." }` instead.
-    ImplicitExprEscape(SmolStr),
     /// Implicit `__entity` escape, in which case we'll see just the TypeAndId
     /// structure
     ImplicitEntityEscape(TypeAndId),
+
+    /// Implicit catch-all case for error handling
+    FoundValue(serde_json::Value),
 }
 
 /// Serde JSON format for Cedar values where we know we're expecting an
 /// extension value
 #[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize)]
 #[serde(untagged)]
 pub enum ExtnValueJson {
-    /// Explicit `__expr` escape; see notes on `CedarValueJson::ExprEscape`.
-    ///
-    /// Deprecated since the 1.2 release; use
-    /// `{ "__extn": { "fn": "...", "arg": "..." } }` instead.
-    ExplicitExprEscape {
```

Comment:
For `EntityUidJson` we left this here reserved, but for `ExtnValueJson` we removed it.  Should we be consistent?

Reply:
yeah